updating/preparing_for_updates/updating-cluster-prepare-past-4.18.adoc

```diff
@@ -0,0 +1,123 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="updating-cluster-prepare-past-4.18"]
+= Preparing to update from {product-title} 4.18 to a newer version
+include::_attributes/common-attributes.adoc[]
+:context: updating-cluster-prepare-past-4.18
+
+toc::[]
+
+Learn about some of the specific concerns around {op-system-base} compute machines before attempting to update from {product-title} 4.18 to a newer version.
+
+[id="migrating-workloads-to-different-nodes_{context}"]
+== Migrating workloads off of package-based {op-system-base} worker nodes
+
+With the introduction of {product-title} 4.19, package-based {op-system-base} worker nodes are no longer supported. Attempting to update your cluster while having these nodes up and running will prevent you from successfully completing the update.
```

Comment:
```suggestion
With the introduction of {product-title} 4.19, package-based {op-system-base} worker nodes are no longer supported. If you try to update your cluster while those nodes are up and running, the update will fail.
```
Not sure if "the update will fail" is too harsh. Feel free to take or ignore this suggestion.